packages/report-server/src/__tests__/reportBuilder/transform/mergeRows.test.ts

```diff
@@ -7,38 +7,39 @@ import {
   MERGEABLE_ANALYTICS,
   UNIQUE_MERGEABLE_ANALYTICS,
   MERGEABLE_ANALYTICS_WITH_NULL_VALUES,
-  MULTIPLE_ANALYTICS,
+  SINGLE_MERGEABLE_ANALYTICS,
 } from './transform.fixtures';
 import { buildTransform } from '../../../reportBuilder/transform';
 
-describe('groupRows', () => {
-  it('throws error when mergeUsing is no specified', () => {
+describe('mergeRows', () => {
+  it('throws error when mergeUsing contains an invalid merge strategy', () => {
     expect(() =>
       buildTransform([
         {
-          transform: 'groupRows',
-          by: 'period',
+          transform: 'mergeRows',
+          using: 'invalid_strategy',
         },
       ]),
     ).toThrow();
   });
 
-  it('throws error when mergeUsing contains an invalid merge strategy', () => {
-    expect(() =>
-      buildTransform([
-        {
-          transform: 'groupRows',
-          mergeUsing: 'invalid_strategy',
-        },
-      ]),
-    ).toThrow();
+  it('defaults to single when using is not specified', () => {
+    const transform = buildTransform([
+      {
+        transform: 'mergeRows',
+        groupBy: 'period',
+      },
+    ]);
+    expect(transform(SINGLE_MERGEABLE_ANALYTICS)).toEqual([
+      { period: '20200101', BCD1: 4, BCD2: 4, BCD3: 4 },
+    ]);
   });
 
-  it('when no by is specified, group to a single row', () => {
+  it('when no groupBy is specified, group to a single row', () => {
```

Comment:
I really like the term `mergeRows` now that neither of the keys are strictly required - it means that you could have

| Doctors | Nurses | Midwives |
|---------|--------|----------|
| 1       |        |          |
|         | 5      |          |
|         |        | 2        |

Then run a simple `mergeRows` and get

| Doctors | Nurses | Midwives |
|---------|--------|----------|
| 1       | 5      | 2        |

Reply:
Yeah me too! I think I had been a bit worried in the past about having a default for `mergeRows` since someone might accidentally omit the `using` and then it would work fine but the data would get messed up (lets say if `last` were the default). 

But now that we have `single` it's actually quite a strict case! And it's kind of natural to just say 'merge those rows' without needing to specify the merge strategy since it's sort of obvious!